src/hll/mod.rs

```diff
@@ -69,8 +69,8 @@ mod list;
 mod serialization;
 mod sketch;
 
-pub use estimator::NumStdDev;
-pub use sketch::HllSketch;
+pub use self::estimator::NumStdDev;
+pub use self::sketch::HllSketch;
```

Comment:
What's the idea behind this? Do we have tools to enforce this style repo-wide?

Reply:
I found it when a mod name sometimes conflicts with a dependency name. Using `self::sketch::HllSketch` always unambiguously refers to the proper symbol.

> Do we have tools to enforce this style repo-wide?

Not quite easy. One can define a custom clippy rule or work with ast-grep. But this rule depends on whether the (pub use) symbol is within this crate/module.